index.html

```diff
@@ -89,5 +93,147 @@ <h3 class="subtitle">I am a Software Developer</h3>
         </li>
       </ul>
     </section>
+    <!--part II-->
+    <section class="workSection">
+      <h1 class="ptitle t2">My Recent Works</h1>
+      <div class="multipost">
+        <img class="divisory" src="./icons/Line.png" alt="divisory line" />
+        <div class="recentwork">
+          <img
+            class="mainpjimg"
+            src="./icons/mainproject.png"
+            alt="My Latest Project"
+          />
+          <h2 class="ptitle t1">Multi-Post Stories</h2>
```

Comment:
- [ ] Good job designing your project cards 💯 Kindly use the correct font size and color for this heading here and also align it to the left as shown in the Figma template 👍 
- Your design:
![Annotation 2022-03-19 062916](https://user-images.githubusercontent.com/85113567/159108546-a3c05f04-d3d5-4aa3-98f1-2d371bd53d95.jpg)
- Figma design:
![Annotation 2022-03-19 062929](https://user-images.githubusercontent.com/85113567/159108552-85b198d5-d1a2-4bd2-aeb9-8752703bf8ac.jpg)

Reply:
just aligned text to the left according to to the design!